helm/resource_release.go

```diff
@@ -890,6 +887,7 @@ func resourceHelmReleaseImportState(d *schema.ResourceData, meta interface{}) ([
 
 	d.Set("name", r.Name)
 	d.Set("description", r.Info.Description)
+	d.Set("chart", r.Chart.Metadata.Name)
```

Comment:
I ran a lint tool on this and it mentioned the error return value isn't being checked for this one. It could be fine. But I'll post the output here just in case:

```
[dakini@dax terraform-provider-helm ((cbb5efb...))]$ golangci-lint run
helm/resource_release.go:888:7: Error return value of `d.Set` is not checked (errcheck)
        d.Set("name", r.Name)
             ^
helm/resource_release.go:889:7: Error return value of `d.Set` is not checked (errcheck)
        d.Set("description", r.Info.Description)
             ^
helm/resource_release.go:890:7: Error return value of `d.Set` is not checked (errcheck)
        d.Set("chart", r.Chart.Metadata.Name)
             ^
helm/resource_release.go:831:5: S1003: should use !strings.Contains(name, "/") instead (gosimple)
        if strings.Index(name, "/") == -1 && repository != "" {
           ^
helm/provider.go:462:27: S1019: should use make([]string, len(s)) instead (gosimple)
        result := make([]string, len(s), len(s))
```

Reply:
Yeah - there is a bunch of linter problems with this repo but it shouldn't be a problem for this particular bug. We should totally clean these up though.